app/controllers/manage/projects_controller.rb

```diff
@@ -1,13 +1,17 @@
 # frozen_string_literal: true
 
+require "csv"
```

Comment:
100% could be wrong but i thought it was no longer necessary to require csv? Is that the case or was I dreaming?

Reply:
I know it's needed in `irb` and _usually_ a gem has loaded it already but I'm not sure if Rails guarantees it. I'll merge but look into it.